extensions/2.0/Vendor/EXT_feature_metadata/schema/featureClass.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Feature class",
+  "type": "object",
+  "description": "A feature class defining the properties that distinguish one set of features from another.",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "name": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The name of the feature class, e.g. for display purposes. The string is UTF-8 encoded."
+      },
+      "description": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The description of the feature class. The string is UTF-8 encoded."
+      },
+      "properties": {
+          "type": "object",
+          "description": "A dictionary, where each key is a property ID and each value is an object defining the property. TODO: do property IDs need to be unique globally, or can they be scoped with the feature class ID?",
```

Comment:
Just trying to make sure TODOs surface to this PR discussion -- how unique should property IDs be?

Reply:
> TODO: do property IDs need to be unique globally

Having thought about this more since writing this TODO, property IDs do not need to be unique globally, just unique within the feature class (which is enforced by being a dictionary)